REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.4
-Compat 0.8.0
+Compat 0.9.5
```

Comment:
`Symbol` is older I believe - probably https://github.com/JuliaLang/Compat.jl/commit/2c11855a34bd2a9e277f5ddbec8eed49df48a02b

Maybe an earlier version of this was using one of the show rewrites? doesn't look like the final version is though